weave

```diff
@@ -735,6 +736,10 @@ echo_addresses() {
     echo $1 $2 $3
 }
 
+echo_just_ips() {
```

Comment:
There's another in progress PR (#1379) that adds an `echo_ips` function that strips the netmask length from the end. Perhaps `echo_just_ips` should be called `echo_cidrs` instead? Any if you want to go with `echo_just_cidrs` I can update the other PR to be `echo_just_ips`...